src/tiered/tiered_handle.c

```diff
@@ -472,13 +473,14 @@ __tiered_open(WT_SESSION_IMPL *session, const char *cfg[])
     /* Collapse into one string for later use in switch. */
     WT_ERR(__wt_config_merge(session, tiered_cfg, NULL, &config));
 
-    WT_ERR(__wt_config_getones(session, config, "key_format", &cval));
-    WT_ERR(__wt_strndup(session, cval.str, cval.len, &tiered->key_format));
-    WT_ERR(__wt_config_getones(session, config, "value_format", &cval));
-    WT_ERR(__wt_strndup(session, cval.str, cval.len, &tiered->value_format));
-    WT_ERR(__wt_buf_fmt(session, tmp, ",tiered_storage=(bucket=%s,bucket_prefix=%s)",
-      tiered->bstorage->bucket, tiered->bstorage->bucket_prefix));
```

Comment:
Question - if the `tiered->key_format` etc. does not get set here, then where?  I don't see that field set in `tiered_config.c`.

Reply:
Thanks for calling that out. I was thinking we didn't need it because we could extract it from the saved config string (and I'd forgotten to remove it from the struct). But on further reflection, I will restore these lines because all the other methods keep a copy of the key/value format. It may be a performance gain.